IBAnimatableTests/AnimatableButtonTests.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  AnimatableButtonTests.swift
+//  IBAnimatable
+//
+//  Created by Steven on 5/12/17.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import XCTest
+@testable import IBAnimatable
+
+final class AnimatableButtonTests: XCTestCase {
+
+  var animatableButton: AnimatableButton!
+
+  override func setUp() {
+    super.setUp()
+    animatableButton = AnimatableButton()
+  }
+
+  override func tearDown() {
+    super.tearDown()
+  }
+
+}
+
+// MARK: - CornerDesignable Tests
+
+extension AnimatableButtonTests: CornerDesignableTests {
+
+  func testCornerRadius() {
+    _testCornerRadius(animatableButton)
+  }
+
+  func test_cornerSides() {
```

Comment:
In IBAnimatable we have a few internal properties using the _ prefix.
These are mainly the the `String` properties used in Interface Builder.

My current approach is to name tests based on the properties/methods they test:

For example:
- `cornerSides` property -> `testCornerSides()`
- `_cornerSides` property -> `test_cornerSides()`

You will also see the _ prefix is the methods provided in the extensions for the test protocols:

For example:
```swift
protocol CornerDesignableTests {
   func testCornerSides()
} 

extension CornerDesignableTests {
   func _testCornerSides() {}
}
```

I'm using this prefix here so it's really clear which method should be called inside the stub when conforming to this protocol.